KasianenkoTymur/HW5/task5_2.py

```diff
@@ -0,0 +1,12 @@
+answer='yes'
+while answer=='yes' or answer=='y':
+    amount=int(input('Enter the number to print the Fibanacci numbers: '))
+
+    for i in range(amount):
```

Comment:
good work, but the formula is complicated and not obvious,
here is a simpler implementation:

n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')